src/test/java/Homework16.java

```diff
@@ -0,0 +1,26 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221778179-b9a0b4ca-8789-4a31-858b-3ccfc4610880.png)

Hint

```
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
```